lib/import/tabular_basic.rb

```diff
@@ -0,0 +1,37 @@
+module Import
+  module TabularBasic
+    private 
+      # valid header fields
+      def parse_headers(row)
+        headers = []
+        $i = 0
```

Comment:
Ruby uses the `$` prefix to denote global variables, which this is not. Is there an enumerable method that could be used here instead?